Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelUploadTests.swift

```diff
@@ -0,0 +1,130 @@
+import NetworkService
+import XCTest
+
+final class SitterProfileViewModelUploadTests: XCTestCase {
+    private var viewModel: SitterProfileViewModel!
+
+    override func setUp() {
+        super.setUp()
+        URLProtocol.registerClass(MockURLProtocol.self)
+        viewModel = SitterProfileViewModel()
+    }
+
+    override func tearDown() {
+        URLProtocol.unregisterClass(MockURLProtocol.self)
+        viewModel = nil
+        super.tearDown()
+    }
+
+    func testUploadSuccessfullyExecuted() async {
```

Comment:
Your class is called `Sitter Profile ViewModel Upload Tests ` but you test the behavior of all the view model (`isErrorOccured`).
Does it make sense?

Reply:
Ok, tried renaming, fixed with 06c4f69